pages/common/xargs.md

```diff
@@ -10,6 +10,10 @@
 
 `{{arguments_null_terminated}} | xargs -0 {{command}}`
 
-- example: list unneeded packages with deborphan and remove them with apt-get
+- example: delete all files that start with 'M'
 
-`sudo deborphan | xargs sudo apt-get remove`
+`ls M* | xargs rm`
+
+- creating variables: copy all files that start with 'M' to the Desktop
```

Comment:
@igorshubovych  It's a note about what the `-J` flag is actually doing. It lets people know that you don't have to use the `%` symbol, but any character will do.